settings/template.ini

```diff
@@ -72,6 +72,10 @@ Methods[]=ftps
 Methods[]=webdav
 Methods[]=webdavs
 
+# Add support for text which is started from www. without protocol.
+# For example change text "www.ez.no" to link tags.
+ChangeUrlWithoutProtocol=disabled
```

Comment:
We think that this setting name should be more clear and specific on what it represents. LinkWwwUrlsWithoutProtocol would be better because it is very clear and describes exactly what your feature does. Your current setting name is too vague (Re: use of terms Change and Url (singular)) and technically inaccurate as it only links urls starting with 'www.'.